src/components/MovieTile/MovieTile.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './MovieTile.css'
+
+export default function MovieTile({ movieImageUrl, movieName, releaseDate, genres, onclickEvent }) {
+
+    const handleGenreClick = (movie) => {
+        onclickEvent(movie)
+    };
+
+    return (
+
+        <div className="mt-card" onClick={handleGenreClick}>
+            <img src={movieImageUrl} alt={movieName} className="mt-image" />
+            <div className="mt-name">{movieName}</div>
+            <div className="mt-release-data">{releaseDate}</div>
+            <div className="mt-gender"> {genres && (
```

Comment:
nit: update genres array, and use the id as the `key`. It's considered a bad practice to use the index as a key. Let's consider that this key is used by React to check what exactly needs to be updated. If one of these items is removed or sorted, this might cause either bug or performance issues